themes/default/content/docs/guides/self-hosted/eks-hosted.md

```diff
@@ -0,0 +1,81 @@
+---
+title: EKS-Hosted Install
+menu:
+    userguides:
+        parent: self_hosted
+        identifier: self_hosted_eks_hosted_install
+        weight: 30
+meta_desc: Installer for deploying the self-hosted Pulumi service in EKS.
+---
+
+The [EKS-Hosted Installer](https://github.com/pulumi/pulumi-self-hosted-installers/eks-hosted) installer is used to deploy the self-hosted Pulumi service in Amazon Elastic Kuberenetes Service (EKS).
+
+## Prerequisites
+
+The customer is required to provide and manage the following:
+
+* Route53 hosted zone.
+
+## EKS-Hosted Deployment
+
+The EKS-hosted installation of Pulumi deploys the following services:
+
+* EKS - Managed Amazon Elastic Kubernetes Service Cluster
+* RDS Aurora - Managed MySQL DB for persistent state and automated replication and snapshotting.
+* S3 - Object storage for checkpoints and policy packs.
+* CloudWatch Logs - Centralized logging for all cluster pods.
+* Route53 - Managed DNS records.
+* ALB - Managed L4/application traffic and TLS termination.
+* ACM - Managed Public TLS certificates.
+
+### Pulumi deploying Pulumi
+
+This installer uses Pulumi to deploy the Pulumi service.  
+In this case, one uses the pulumi CLI with a self-managed backend (e.g. S3) to deploy all services listed above to stand up the self-hosted Pulumi Service.  
+The installation package includes Pulumi project code so that you can deploy the service by running `pulumi up`.
```

Comment:
```suggestion
This installer uses Pulumi to deploy the Pulumi Service. In this case, one uses the Pulumi CLI with a self-managed backend (e.g. S3) to deploy all services listed above to stand up the self-hosted Pulumi Service. The installation package includes Pulumi project code so that you can deploy the Service by running `pulumi up`.
```

Reply:
pushing an update to fix this